Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Tizen.Uix.Tts;
+
+namespace Xamarin.Essentials
+{
+    public static partial class TextToSpeech
+    {
+        static TtsClient tts = null;
+        static TaskCompletionSource<bool> tcsInitialize = null;
+        static TaskCompletionSource<bool> tcsUtterances = null;
+
+        internal static async Task PlatformSpeakAsync(string text, SpeechOptions options, CancellationToken cancelToken = default)
+        {
+            await Initialize();
+
+            if (tcsUtterances?.Task != null)
+                await tcsUtterances.Task;
+
+            tcsUtterances = new TaskCompletionSource<bool>();
+            if (cancelToken != null)
+            {
+                cancelToken.Register(() =>
+                {
+                    tts?.Stop();
+                    tcsUtterances?.TrySetResult(true);
+                });
+            }
+
+            var language = "en_US";
+            var voiceType = Voice.Auto;
+            if (options?.Locale.Language != null)
+            {
+                foreach (var voice in tts.GetSupportedVoices())
+                {
+                    if (voice.Language == options.Locale.Language)
+                    {
+                        language = voice.Language;
+                        voiceType = voice.VoiceType;
+                    }
+                }
+            }
+
+            var pitch = tts.GetSpeedRange().Normal;
+            if (options?.Pitch.HasValue ?? false)
+                pitch = (int)(options.Pitch.Value / PitchMax * tts.GetSpeedRange().Max);
```

Comment:
Rounding convention is required to this value.

Reply:
done